pkg/apis/sources/v1beta1/kafka_types.go

```diff
@@ -67,6 +67,11 @@ type KafkaResourceSpec struct {
 
 // KafkaSourceSpec defines the desired state of the KafkaSource.
 type KafkaSourceSpec struct {
+	// Number of desired replicas. This is a pointer to distinguish between explicit
+	// zero and not specified. Defaults to 1.
+	// +optional
+	Replicas *int32 `json:"replicas,omitempty"`
```

Comment:
Doing it on `v1`? instead ? :grimacing:

Reply:
We should allow adding new fields to an existing API without changing the version. Kubernetes allows it. See https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#adding-unstable-features-to-stable-versions

Excerpt: `.. It is generally allowed to add new fields without changing the API version, ..`